src/AddGroceries.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { db } from './lib/firebase';
+
+function AddGroceries(props) {
+  return (
+    <div>
+      {/* <form id="test-form" onSubmit={submitItem}> */}
+      <form id="test-form">
+        {/* <input type="text" value={groceryItem} /> */}
+        <input type="text" />
+        <input type="submit" />
+      </form>
+    </div>
+  );
+}
+
+// function submitItem() {
+//     db.collection("items").add({
+//         input: groceryItem
+//     })
+//     .then((docRef => {
+//     console.log("Document written with ID: ", docRef.id);
+//     })
+//     .catch((error) => {
+//     console.error("Error adding document: ", error);
+// })
+// }
```

Comment:
For future PRs, it is okay to have functionality and other piece uncommented for us to try it out. I think we may have forgotten to mention this, but you probably noticed that when you made the PR, there was a comment left by Github Actions that actually links to a preview of the work that you've done on your branch so that anyone who views the PR can quickly review it. I would say that the tasks for reading and writing to FireStore can be marked complete once it can be successfully executed with the preview page.

Regarding the code, once I uncommented it and moved it into the component function, it worked! One thing to remember is that when you submit a form, it reloads the page by default, so there needs to be an argument `e` passed to the function, and a call to `e.preventDefault()` at the top.